src/router/routes/errorHandling.ts

```diff
@@ -22,10 +23,22 @@ export const applicationErrorMapper = (
         next(new createError.BadRequest(`Failed to parse request body as JSON resource. Error was: ${err.message}`));
         return;
     }
+    if (err instanceof TooManyConcurrentExportRequestsError) {
+        next(
+            new createError.TooManyRequests(
+                'There is currently too many concurrent export requests. Please try again later',
```

Comment:
Lets make this generic
```suggestion
                'There is currently too many requests. Please try again later',
```

Reply:
Updated